lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -50,15 +51,15 @@ sub index : Chained('root') PathPart('') Args(0) {
     my ( $author, $data ) = ( $author_cv->recv, $releases_cv->recv );
     $c->detach('/not_found') unless ( $author->{pauseid} );
 
-    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{user} );
+    my $faves_cv = $c->model('API::Favorite')->by_user( $author->{pauseid} );
```

Comment:
I didn't find the `user` entry in `$author`. it was a hack to make it work - but I agree it may have been a wrong choice.
Will have another look.